### PR TITLE
Bookstore styling

### PR DESCRIPTION
**Pull request by inspecta**
- Add classes to component elements
- Add styles to match Zeplin's design

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
 *::after {
   box-sizing: border-box;
   padding: 0;
   margin: 0;
+  font-family: "Circular Book", sans-serif;
+
+  /* font-family: "Circular Medium", sans-serif;
+  font-family: "Circular Bold", sans-serif;
+  font-family: "Montserrat", sans-serif;
+  font-family: "Roboto Slab", serif; */
```
Comment (OLIPLICHE) [inline src/index.css]: - Please keep your CSS code clean.  Remove the commented code,

File: src/Components/AddBook.js [hunk 2]
```diff
@@ -36,23 +36,29 @@ const AddBook = () => {
...
+            type="text"
+            name="title"
+            className="input title"
+            placeholder="Enter Title"
+            value={state.title}
+            onChange={onChange}
+            required
+          />
+          <input
+            type="text"
```
Comment (OLIPLICHE) [inline src/Components/AddBook.js]: - Good job so far 👍 
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @inspecta, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- App is working fine
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @inspecta :wave:

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:


## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following GitFlow
✔️ Following requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**